frontend/javascripts/admin/auth/registration_form_wkorg.tsx

```diff
@@ -15,34 +17,34 @@ type Props = {
   onRegistered: (isUserLoggedIn: true) => void;
 };
 
-function generateOrganizationId() {
-  let output = "";
-
-  for (let i = 0; i < 8; i++) {
-    output += Math.floor(Math.random() * 255)
-      .toString(16)
-      .padStart(2, "0");
-  }
+function generateOrganizationId(firstName: string, lastName: string) {
+  return `${firstName.toLowerCase()}-${lastName.toLowerCase()}-lab`;
+}
```

Comment:
I think this might create naming collisions. I'd like to double check this with @fm3

Reply:
And this way of defining the orga id might lead to orga ids with whitespaces in it :thinking:

Reply:
Why change this in this PR at all? As I understand, the random-character string is supposed to be the typical organization id. The organization name on the other hand is supposed to be created from the user name. 

This was introduced in https://github.com/scalableminds/webknossos/pull/4439 (though organizationId was still called organizationName then, and what is now the name was then the displayName).

I agree that changing it to firstname-lastname-lab may lead to unwanted collisions.

Reply:
To me it is also kinda strange, why the frontend generates this id string and not the backend :thinking:. Maybe this could be changed in the future?

> but it is indeed intended that new ones created by this form should have the random string.

@dieknolle3333 Could you please undo this change here? In case we agree to change the id generation, I'd do this in a separate PR. But I'd say that this is kinda unlikely because an id being a random hex string sounds fine to me. But feel free to open up a discussion on this :) (@frcroth)

Reply:
It is undone. I interpreted the PR and the issue so that the id should be human readable now.

Reply:
Oh that indeed looks like a misunderstanding :see_no_evil: 

Thanks a lot for undoing the change so fast :) :pray:

Reply:
> To me it is also kinda strange, why the frontend generates this id string and not the backend 🤔. Maybe this could be changed in the future?

Wrote #8216 to track that and future discussion of what the IDs should look like.

> I interpreted the PR and the issue so that the id should be human readable now.

Fair, but it was more of the opposite, really. Before #7386 there was a mongoId *and* this random string as organizationName, *and* the actual name (then called displayName), with spaces and all. In the PR, we dropped the mongoId and declared this random string the id, because it already had to be unique and was used as a key in many places.

Thanks to you all for clearing this up here :)